src/peer/media/sender/component.rs

```diff
@@ -611,6 +622,40 @@ impl Component {
         }
         Ok(())
     }
+
+    /// Disables media exchange on a local track acquisition error.
+    #[watch(self.send_encodings.subscribe().skip(1))]
+    async fn send_encodings_updated(
+        sender: Rc<Sender>,
+        _: Rc<State>,
+        enc_params: Guarded<Vec<proto::EncodingParameters>>,
+    ) -> Result<(), Traced<RtcPeerConnectionError>> {
+        let (enc_params, _guard) = enc_params.into_parts();
+
+        let target_codecs = probe_target_codecs(
+            enc_params.iter().filter_map(|e| e.codec.as_ref()),
+        )
+        .await;
+        // TODO: Switch to negotiationless codec change via
+        //       RTCRtpSender.setParameters when
+        //       RTCRtpEncodingParameters.codec is supported on all
+        //       major UAs.
+        if let Some(target_codecs) = target_codecs {
```

Comment:
@alexlapa judging from here, it seems to be supported enough:
https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpSender/setParameters#browser_compatibility

Reply:
@tyranron ,

We need [RTCRtpEncodingParameters.codec](https://www.w3.org/TR/webrtc/#dom-rtcrtpencodingparameters-codec) specifically. That only works in Chrome atm and is completely missing from the compatibility table (`parameters.encodings.codec`).